### ISSUE TITLE
how to generate "data.npy" and "label.npy"

### ISSUE THREAD

**Issue by Aysha9** [amobiny/3D-AlexNet-Tensorflow-Tensorboard#1]
Hi, 
First of all, I would like to precise that I am new to Python and to the use of neural networks algorithms. 

I can't understand how to generate the files 'Data.npy' and 'Label.npy'. More precisely, I could not understand the description of the file which is "each input set must be: (#images, dim1, dim2, dim3, 1)". In my comprehension #is used for comments, but here is it the whole image ?
Do I have to concatenate all the images in this file ? 

I have the same question for the description of ''label.npy" which is "The corresponding label must be: (#images, #classes)". I can't understand the format of the data inside the file.  
 
Many thanks for your response !!
